pkg/sippyserver/server.go

```diff
@@ -66,9 +66,10 @@ type TestGridDashboardCoordinates struct {
 }
 
 type StandardReport struct {
-	CurrentPeriodReport sippyprocessingv1.TestReport
-	CurrentTwoDayReport sippyprocessingv1.TestReport
-	PreviousWeekReport  sippyprocessingv1.TestReport
+	CurrentPeriodReports []sippyprocessingv1.TestReport
+	CurrentPeriodReport  sippyprocessingv1.TestReport
+	CurrentTwoDayReport  sippyprocessingv1.TestReport
+	PreviousWeekReport   sippyprocessingv1.TestReport
```

Comment:
seems like this should also be a slice so that you get the correct, "seven days before the prior interval", right?

Reply:
yeah I guess we should compute the rolling prior 7 days